site/docs/how-tos/custom-batch-source.md

```diff
@@ -1,28 +1,25 @@
 ---
-title: Create a Batch Source
+title: Add Batching to Custom Source
```

Comment:
title seems wrong, it's about creating a batch source, not adding batching to a source

Reply:
Actually the only new point this How-To introduces is the batching. Originally there was a tutorial, then it got split into the "basics" which are in _Sources and Sinks_ and this how-to, which explains how to add batching.

Reply:
ok, but it's confusing if user is trying to look for how to create a custom source, not just add batching to it, where 's the entry point?

Reply:
In the section titled "Sources and Sinks", subsection "Custom Sources and Sinks".